src/graph/test/SchemaTest.cpp

```diff
@@ -45,10 +45,10 @@ TEST_F(SchemaTest, metaCommunication) {
         };
         ASSERT_TRUE(verifyResult(resp, expected));
     }
-    // test nonexistent space
     {
+        // test nonexistent space
         cpp2::ExecutionResponse resp;
-        std::string query = "USE default_space";
+        std::string query = "USE not_exist_space";
         auto code = client->execute(query, resp);
         ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```

Comment:
Why return succeeded if the space not exist?

Reply:
using `E_EXECUTION_ERROR` to check return code and avoid `ASSERT_NE`

Reply:
A HA,  my bad.